### ISSUE TITLE
Add viewer for clean birds and bicycles dataset

### ISSUE THREAD

**Issue by nottombrown** [google/unrestricted-adversarial-examples#27]
Currently we save each image into their own files. It would be nice to run a local webserver that renders them all into a single page so that people can inspect them and verify that they are all unambiguous.

I have a messy prototype that I set up for SVHN (adapted from a colab notebook)

https://gist.github.com/nottombrown/783b5553823f4c1b35395fdc144b96fa

![image](https://user-images.githubusercontent.com/306655/45790400-811b2e00-bc39-11e8-955a-90ca6c228364.png)

### Rough spec
- [ ] In the `evaluate_two_class_unambiguous_model` function, dump a map from `image_id => (logits, confidence, labels)` to a `.json` file in the `plot_confident_error_rate` directory
- [ ] Set up a minimal flask or node server that parses the `json` file 
- [ ] Set up the server to [render the viewer](https://gist.github.com/nottombrown/783b5553823f4c1b35395fdc144b96fa) using svelte (this should work out of the box)
- [ ] Add installation instructions
